website/docs/reference/api/legacy/unleash/admin/segments.mdx

```diff
@@ -4,9 +4,9 @@ title: /api/admin/segments
 
 import ApiRequest from '@site/src/components/ApiRequest'; export const basePath = "api/admin/segments"; export const path = (p) => `${basePath}/${p}`;
 
-:::info Availability
+:::note Availability
 
-Segments are available to Unleash Pro and Unleash Enterprise users since **Unleash 4.13**.
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing) | **Version**: `4.13+`
```

Comment:
does it make sense to have both links be the same? We probably don't have separate links for them, but it feels kinda redundant 🤔

Reply:
Yeah, we just have the one link. I didn't really have a better idea formatting-wise than to link both